scripts/upgrade-ssv-network.ts

```diff
@@ -1,5 +1,7 @@
+import { ethers, upgrades } from 'hardhat';
+
 async function upgradeSSVNetwork() {
-  const proxyAddress = process.env.SSVNETWORK_PROXY_ADDRESS;
+  const proxyAddress: any = process.env.SSVNETWORK_PROXY_ADDRESS;
```

Comment:
Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)